metadata/location/utah_address_points

```diff
@@ -0,0 +1,85 @@
+# Title
+
+Utah Address Points
+
+## ID
+
+4373453d-9e20-44da-9c6e-1f0b1ea653af
+
+## Brief Summary
+
+Contains US Postal Service address points for all twenty-nine counties in Utah.
+
+## Summary
+
+An address point represents a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail.
+
+## Description
+
+### What is the dataset
+
+This dataset contains vector point data for over one million addresses in the State of Utah and includes information such as the full street address and the type of unit the point represents. This dataset is a mapping-grade framework layer that receives regular updates. Maintaining this dataset is an ongoing effort between UGRC and county-level data stewards.
+
+### What is the purpose of the dataset
+
+Address data are used for a variety of purposes, including [geocoding](api.mapserv.utah.gov), city planning, and [redistricting](https://redistricting.utah.gov/wp-content/uploads/2021/07/Redistricting-Packet-1-compressed.pdf). In addition, UGRC actively collaborates with the US Postal Service and Google Maps to improve mail and package delivery, navigation apps, and emergency services.
+
+### What does the dataset represent
+
+Points are generally placed on the structure of the address, but may also be found on the driveway or otherwise within the parcel of land. In cases where the county has provided the specific location of the address point for a given address, it is denoted in the "PtLocation" field. The address points in this dataset also include information such as:
+
+- Full street address (i.e., the USPS mailing address, if the address is for a physical location [rather than a PO box])
+- Whether the location is a building
+- The type of unit
+- The city and ZIP code
+- Unique code identifiers, including the Federal Information Processing Standard ([FIPS](https://www.nist.gov/standardsgov/compliance-faqs-federal-information-processing-standards-fips)) county code and the US National Grid ([USNG](https://www.fgdc.gov/usng/how-to-read-usng)) spatial address
+- The date that the address point was added to the dataset
+
+### How was the dataset created
+
+The information in the Address Points dataset was originally compiled by Utah counties and was aggregated by UGRC for the Master Address List (MAL) grant initiative in 2012. The purpose of this initiative was to ensure that all state entities were using the same verified, accurate county and municipal address information.
```

Comment:
I think we should also have how it is updated (ie, we get MALs from counties and integrate them into the dataset, if I remember correctly). Otherwise, I don't see any info on how it's being updated.

Reply:
I've added that in. Other than saying that we receive the updates and then aggregate them, are there any other details you think ought to be included?